boards/matrix_portal_m4/README.md

```diff
@@ -23,4 +23,15 @@ $ cargo hf2 --release --example blinky_basic --vid 0x239a --pid 0x00c9
     Flashing "/Users/User/atsamd/boards/matrix_portal_m4/target/thumbv7em-none-eabihf/release/examples/blinky_basic"
     Finished in 0.051s
 $
-```
```

Comment:
Looks like the readme got a little messed up and there's duplicate blocks here.

Reply:
Fixed tacks